.github/CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 <h3>New features since last release</h3>
 
+* "compact" decompositions as described in <https://arxiv.org/abs/2104.07561>,
```

Comment:
This feels like it should be capitalized, even though it's using quotation marks?
```suggestion
* Compact decompositions as described in <https://arxiv.org/abs/2104.07561>,
```